src/app/showcase/modal-showcase/modal-showcase.component.ts

```diff
@@ -43,5 +43,11 @@ export class ModalShowcaseComponent {
       defaultValue: '',
       inputValues: ['Component'],
     },
+    {
+      name: 'componentProps',
+      description: 'The data to pass to the modal component.',
+      defaultValue: '',
+      inputValues: ['undefined | { [key: string]: any; }'],
```

Comment:
Isn't the the undefined type redundant?

Reply:
You can argue that it is, however I copied this from Ionic's documentation and personally it indicates for me that it's "optional".

Reply:
You can argue that it is, however I copied this from Ionic's documentation and personally it indicates for me that it's "optional".